the-collab-lab/tcl-76-smart-shopping-list
# Add more keys and unique key props

## Description

- One of the problems was that` list.id` and `item.id` either didn't exist or were identical to item name (so not unique enough)
- Another was that the properties on `filteredObject` were mapped through and no assigned keys.
- Used `crypto.randomUUID()` in the rendering in `Home.js` , and added a Fragment in `List.jsx` to add the key to the outside mapping.

## Related Issue

closes issue #32 

## Acceptance Criteria

- [x] The `key` prop is added to the appropriate list element
- [x] The console is clear of any errors

## Type of Changes

`bug fix`

## Updates

- Used `crypto.randomUUID()` for all keys
- Added a new key for the outer map on `filteredObject` in `List.jsx`

## Testing Steps / QA Criteria

- Click through your lists and view both the Home page and List Page
- There should be no errors in the console for invalid Key not being unique.
